fdbserver/ptxn/TLogServer.actor.cpp

```diff
@@ -1596,6 +1596,7 @@ ACTOR Future<Void> commitPeekAndCheck(std::shared_ptr<test::TestDriverContext> p
 	state Version endVersion(beginVersion + deterministicRandom()->randomInt(5, 20));
 	state Optional<UID> debugID(test::randomUID());
 
+/*
```

Comment:
Are you going to fix this in the next PR?

Reply:
Yes, this is on the way.